pkgs/applications/science/electronics/simulide/default.nix

```diff
@@ -0,0 +1,55 @@
+{ appimageTools, symlinkJoin, lib, fetchurl, makeDesktopItem }:
+
+let
+  pname = "simulide";
+  version = "0.4.15";
+  release = "SR10";
+
+  src = fetchurl {
+    url = "https://launchpad.net/${pname}/${version}/${version}-stable/+download/SimulIDE_${version}-${release}.AppImage";
+    sha256 = "25f85f5262b39c8b25da042f11a0fd517c5d2750e81e538ac15992659ebea03c";
+  };
+
+  binary = appimageTools.wrapType2 {
+    name = pname;
+    inherit src;
+  };
+  # we only use this to extract the icon
+  appimage-contents = appimageTools.extractType2 {
+    name = "SimulIDE_${version}-${release}";
+    inherit src;
+  };
+
+  desktopItem = makeDesktopItem {
+    name = "simulide";
+    exec = "${binary}/bin/simulide";
+    icon = "simulide";
+    comment = "A simple real time electronic circuit simulator, supporting PIC, AVR and Arduino.";
+    desktopName = "SimulIDE";
+    genericName = "SimulIDE";
+    categories = [ "Education" "Electronics" ];
+  };
+
+in
+  symlinkJoin {
+    name = "${pname}_${version}-${release}";
```

Comment:
While trying the package with `nix run`, it tries to run it with the name "simulide_0.4.15-SR10", but the name of the binary is "simulide", which results in an error. The issue is probably this name in symlinkJoin not matching the binary.

Reply:
Hmm, sorry about that. I forgot to execute ```nix run```, so I haven't seen this problem earlier.